<launch>
  
  <node pkg="monocular_pose_estimator" name="command_listener" type="command_listener.py" output="screen" />

  <node pkg="monocular_pose_estimator" name="record_experiment" type="record_experiment.py" />

  <node pkg="rosbag" name="rosbag" type="record" args="/record/desired_pose /record/camera_info /record/image_raw /record/estimated_pose" output="screen" />

</launch>
